src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/UploadEnvelopeDocumentsTaskTest.java

```diff
@@ -84,28 +81,6 @@ void should_call_service_twice_when_2_different_containers_are_present_in_the_li
         verify(uploadService, times(2)).processByContainer(anyString(), anyList());
```

Comment:
Better should be `times(envelopes .size())` instead of `times(2)`

Reply:
It's not about number of envelopes, it's number of containers.